cli/src/fmt/mod.rs

```diff
@@ -17,6 +22,14 @@ pub trait Render<T> {
         None
     }
 
+    fn render_nested(&self, item: &T) -> Option<String> {
```

Comment:
I don't think this should be part of the trait. This feels very particular to the simple format and should probably be implemented on the simple type itself.

If it's implemented in `Simple` then we could even store the ident level in it. That would mean that we can increment when we create the inner renderer. Then nesting is aware of the depth.